src/entity/Bullet.java

```diff
@@ -1,74 +1,69 @@
 package entity;
 
 import java.awt.Color;
-
 import engine.DrawManager.SpriteType;
 
 /**
  * Implements a bullet that moves vertically up or down.
  * 
- * @author <a href="mailto:RobertoIA1987@gmail.com">Roberto Izquierdo Amo</a>
- * 
+ * @author ...
  */
 public class Bullet extends Entity {
 
-	/**
-	 * Speed of the bullet, positive or negative depending on direction -
-	 * positive is down.
-	 */
-	private int speed;
+    /**
+     * Speed of the bullet, positive or negative depending on direction - positive is down.
+     */
+    private double speed;
 
-	/**
-	 * Constructor, establishes the bullet's properties.
-	 * 
-	 * @param positionX
-	 *            Initial position of the bullet in the X axis.
-	 * @param positionY
-	 *            Initial position of the bullet in the Y axis.
-	 * @param speed
-	 *            Speed of the bullet, positive or negative depending on
-	 *            direction - positive is down.
-	 */
-	public Bullet(final int positionX, final int positionY, final int speed) {
-		super(positionX, positionY, 3 * 2, 5 * 2, Color.WHITE);
-
-		this.speed = speed;
+    private static final int FPS = 60;
```

Comment:
You shouldn't need this variable if you assign the speed through the constructor. The FPS is also determined in the Core.java file.